docs/source/examples/text.rst

```diff
@@ -1,12 +1,129 @@
 Text
 =================================
 
-.. manim:: Text1
+.. manim:: Example1Text
     :quality: medium
     :save_last_frame:
 
-    class Text1(Scene):
+    class Example1Text(Scene):
         def construct(self):
-            t = TextMobject("Hello World")
-            self.add(t)
+            text = Text('Hello world').scale(3)
+            self.add(text)
+
+.. manim:: Example2Text
+    :quality: medium
+    :save_last_frame:
+
+    class Example2Text(Scene):
+        def construct(self):
+            text = Text('Hello world', color=BLUE).scale(3)
+            self.add(text)
+
+
+.. manim:: Example3Text
+    :quality: medium
+    :save_last_frame:
+
+    class Example3Text(Scene):
+        def construct(self):
+            text = Text('Hello world', gradient=(BLUE, GREEN)).scale(3)
+            self.add(text)
+
+
+.. manim:: Example4Text
+    :quality: medium
+    :save_last_frame:
+
+    class Example4Text(Scene):
+        def construct(self):
+            text = Text('Hello world', t2g={'world':(BLUE, GREEN)}).scale(3)
+            self.add(text)
+
+.. manim:: Example5Text
+    :quality: medium
+    :save_last_frame:
+
+    class Example5Text(Scene):
+        def construct(self):
+            text = Text('Hello world', font='Source Han Sans').scale(3)
+            self.add(text)
+
+.. manim:: Example5bText
+    :quality: medium
+    :save_last_frame:
+
+    class Example5bText(Scene):
+        def construct(self):
+            text = Text('Hello world', t2f={'world':'Forte'}).scale(3)
```

Comment:
There seems to be a problem with changing the font right now.
Also when I run this locally, without building the docs, Text it is not using the right font...
I would nevertheless keep this example for now, and make an note to it, that this still has to be improved.

Reply:
...or we try to fix the font :)